hugegraph-cluster-test/hugegraph-clustertest-minicluster/pom.xml

```diff
@@ -0,0 +1,66 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
this module is just for testing or can also be used for deployment in production environments?

Reply:
It is designed to be used in production environments